app/assets/stylesheets/provider/admin/apiconfig/services/_proxies.scss

```diff
@@ -220,7 +220,7 @@ form.formtastic fieldset > ol > li.radio--iconic {
     z-index: 1000;
     top: line-height-times(1);
     left: 0;
-    width: 100%;
+    width: 96%;
```

Comment:
hmmm... we don't have any other solution to this? Like a patternfly fn, grid, container, etc... because this changes drastically depending on the screen size.